content/applications/hr/time_off/allocations.rst

```diff
@@ -0,0 +1,124 @@
+
+.. image:: allocations/new-allocation.png
+   :align: center
+   :alt: A new allocation form with all the fields filled out for the annual two week vacation
+         granted to all employees.
+
+.. _time_off/request-allocation:
+
+Request allocation
+==================
+
+If an employee has used all their time off, or is going to run out of time off, they can request an
+allocation for additional time. Allocations can be requested in one of two ways, either from the
+:ref:`Dashboard <time_off/dashboard>` or the :ref:`My Allocations <time_off/my-allocations>` view.
+
+To create a new allocation request, click either the :guilabel:`New Allocation Request` button on
+the main *Time Off* dashboard, or the :guilabel:`New` button in the :guilabel:`My Allocations` list
+view. Both buttons open a new allocation request form.
+
+.. note::
+   Both :guilabel:`New` buttons open a :guilabel:`New Allocation` request form, but when requested
+   from the :guilabel:`Dashboard`, the form appears in a modal. When requested from the
+   :guilabel:`My Allocations` list view, the screen navigates to a new allocation request page,
+   instead.
+
+Enter the following information on the :guilabel:`New Allocation` request form:
+
+- :guilabel:`Time Off Type`: select the type of time off being requested for the allocation from the
+  drop-down menu. After a selection is made, the title updates with the time off type.
+- :guilabel:`Validity Period`: this field **only** appears if requesting an allocation with a time
+  off type that has the :guilabel:`Take Time Off in` field :ref:`configured
+  <time_off/time-off-officer>` to either :guilabel:`Day` or :guilabel:`Half Day`. This field does
+  **not** appear if the :guilabel:`Take Time Off in` field is set to :guilabel:`Hours`.
+
+  Using the calendar selector, select a start and end date the allocation is valid for. By default,
+  the current date populates the first field, and :guilabel:`No Limit` populates the second field.
+
+  Adjust both of these dates, if desired.
+- :guilabel:`Allocation`: enter the amount of time being requested in this field. The format is
+  presented in either days or hours, depending on how the time off type is configured. Once this
+  field is populated, the name of the allocation request is updated to include the amount of time
+  being requested.
+- :guilabel:`Add a reason...`: enter a description for the allocation request. This should include
+  any details that managers and approvers may need in order to approve the request.
+
+If the request was created from the :guilabel:`Dashboard`, click the :guilabel:`Save & Close` button
+to save the information and submit the request.
+
+If the form was completed from the :guilabel:`My Allocations` list view, the information is
+automatically saved as it is entered. However, the form can be saved manually at any time by
+clicking the *save manually* button, represented by a :guilabel:`(cloud upload)` icon.
```

Comment:
```suggestion
clicking the *Save manually* button, represented by a :guilabel:`(cloud upload)` icon.
```
I'd suggest trying to find the FA icon for this save manually mention. Or, since this is one of those 'universal' features -- found throughout the entirety of Odoo -- you might even consider omitting it altogether, as it stands to reason that the reader may already be aware of this element. Just something to think about during your revisions.